test/e2e/framework/pods.go

```diff
@@ -29,70 +29,74 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// TODO: Consolidate pod-specific framework functions here.
-
 // Convenience method for getting a pod client interface in the framework's namespace.
-func (f *Framework) PodClient() unversioned.PodInterface {
-	return f.Client.Pods(f.Namespace.Name)
+func (f *Framework) PodClient() *PodClient {
```

Comment:
I'm not sure I agree with the naming change. I would call this the "PodInterface" (or just "Pods"), and f.Client.Pods(...) the PodClient. Can you explain your decision to invert it?